### ISSUE TITLE
Strange behavior in Sync-BN implementation

### ISSUE THREAD

**Issue by XiaLiPKU** [CSAILVision/semantic-segmentation-pytorch#122]
https://github.com/CSAILVision/semantic-segmentation-pytorch/blob/3b2710f0c6a0a8fd8389ee69cbcda0a298278d3d/lib/nn/modules/batchnorm.py#L132
I just can't figure out why use this formulation.

Here, I see the original implementation in
https://github.com/vacancy/Synchronized-BatchNorm-PyTorch/blob/b998cb3460172c638c403524c9ff384ec507d20b/sync_batchnorm/batchnorm.py#L122
which is easy to understand
